application/src/main/scala/com/azavea/franklin/tile/package.scala

```diff
@@ -0,0 +1,38 @@
+package com.azavea.franklin
+
+import com.amazonaws.services.s3.AmazonS3URI
+import geotrellis.store.s3.util.S3RangeReader
+import geotrellis.util.{FileRangeReader, StreamingByteReader}
+
+import java.net.URI
+import java.nio.file.Paths
+
+package object tile {
+
+  /** Replicates byte reader functionality in GeoTrellis that we don't get
+    * access to
```

Comment:
because it's in `gt-spark`?

Reply:
No - I think this is actually an older way of how GT used to choose/find the right byte reader in various contexts. As far as I know it doesn't necessarily have to do with `gt-spark` -- but for our own reader in here (that creates a tiff from info and reads geotiff info) we need to find a byte reader.

GT now uses some service provider mechanism that I don't fully understand and this is easier for me to grok.